package main;

import customExceptions.RecordAlreadyInDatabaseException;
import database.*;
import operation.*;
import user.*;

import java.io.*;

public class MedicalCentre {
    public static void main(String[] args) throws IOException {
    }
}
